colossalai/inference/core/engine.py

```diff
@@ -57,6 +57,7 @@ def __init__(
         self.tokenizer.pad_token = self.tokenizer.eos_token
         self.generation_config = inference_config.to_generation_config(self.model_config)
         model = model.eval()
+        model = model.cuda()
```

Comment:
Then we might want to remove `.cuda()` in `_shardformer` (duplicated)

Reply:
Done